packages/atomic/src/pages/v1.html

```diff
@@ -231,8 +231,21 @@
                   font-weight: bold;
                   margin-right: 0.25rem;
                 }
+
+                .thumbnail {
+                  display: block;
+                  width: 100%;
+                  height: 100%;
+                }
               </style>
-              <atomic-result-section-visual><atomic-result-icon></atomic-result-icon></atomic-result-section-visual>
+              <atomic-result-section-visual>
+                <atomic-size-condition maximum-width-inclusive="3rem">
+                  <atomic-result-icon></atomic-result-icon>
+                </atomic-size-condition>
+                <atomic-size-condition minimum-width-exclusive="3rem">
+                  <img src="https://picsum.photos/350" class="thumbnail" />
```

Comment:
Couldn't a dev add a `style` attribute on the element they want to adjust, and set the min-width there?
If so, is a web-component needed?

Reply:
I can't get `min-width` to hide an element. Min-width seems to prevent the width from going under a certain value, correcting it rather than hiding it. I saw an article that found a way to do it with flexboxes, but they added that it seems to break accessibility and not work with every type of content.